lte/gateway/configs/mme.yml

```diff
@@ -36,3 +36,4 @@ accept_combined_attach_tau_wo_csfb: true
 
 # Enable IPv6 support for S1AP SCTP endpoint
 s1ap_ipv6_enabled: true
+enable_IMS_VoPS_3GPP: false
```

Comment:
Put it only under mme.conf.template. Inside that only in the AMF section

Reply:
Comment addressed.